### ISSUE TITLE
[Bug]: Wiki/CKAN's RPM repo: Outdated command to add repo on Fedora Linux

### ISSUE THREAD

**Issue by ratemisia** [KSP-CKAN/CKAN#4221]
### Is there an existing issue for this?

- [X] I have checked the existing issues for an existing issue. I even actually looked at the ones that are pinned to the top!

### Operating System

Ultramarine Linux (Fedora)

### CKAN Version

Installing latest version

### Games

KSP 1

### Game Version

N/A (Wiki issue)

### Did you make any manual changes to your game folder (i.e., not via CKAN)?

N/A (Wiki issue)

### Describe the bug

On https://github.com/KSP-CKAN/CKAN/blob/master/doc/rpm-repo.md, the instructions to add CKAN as an rpm repo direct the user to run these commands:
```
sudo dnf config-manager --add-repo https://ksp-ckan.s3-us-west-2.amazonaws.com/rpm/stable/ckan_stable.repo
sudo dnf install ckan
```
However, the --add-repo command has recently changed on Fedora-based distributions. The correct command now is:
```
sudo dnf config-manager addrepo --from-repofile https://ksp-ckan.s3-us-west-2.amazonaws.com/rpm/stable/ckan_stable.repo
sudo dnf install ckan
```

### Steps to reproduce

Running the command that is currently on the wiki results in an error, seen below in "Relevant log output." Running the updated command I supplied above causes CKAN and dependencies to install without incident. This has only changed very recently, so only quite up-to-date Fedora based distributions need the new command. Leaving the old command available for those who haven't updated yet, at least for a while, may be wise.

### Relevant log output

Unknown argument "--add-repo" for command "config-manager". Add "--help" for more information about the arguments.

**Comment by HebaruSan** [comment]
Hey @ratemisia, thanks for the report. Why don't you submit a pull request to fix this? I don't have a RPM-based distro set up for testing at the moment.

**Comment by ratemisia** [comment]
I can work on it! I'll leave this open until I can submit a PR and then close it.

**Comment by HebaruSan** [comment]
Awesome! If you put `fixes #4221` in the description, it'll auto-close upon merging.
